2-mandatory/3-financial-times.js

```diff
@@ -14,7 +17,13 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+  // TODO
+  let shortestTitle = allArticleTitles[0];
+  for (let i = 0; i < allArticleTitles.length; i++) {
+    if (allArticleTitles[i].length < shortestTitle.length)
+      shortestTitle = allArticleTitles[i];
+  }
+  return shortestTitle;
```

Comment:
titleWithFewestWords current fails the test, we can see why on line 23, what does the allArticleTitles[i].length tell you? Does it tell you the number of words in the string or does it tell you something else?

Reply:
words not letters, thanks Becky